File: Source/CslaExtremeDemos.BusinessVB/CivilStatus.vb
Imports System.ComponentModel

Namespace CslaExtremeDemos.BusinessVB
    Public Enum CivilStatus As Byte
        <Description("<Select status>")>
        Empty = 0
        <Description("Single")>
        Singleton
        <Description("Maried")>
        Maried
        <Description("Divorced")>
        Divorced
        <Description("Widowed")>
        Widowed
    End Enum
End Namespace

File: Source/CslaExtremeDemos.Business/CivilStatus.cs
using System.ComponentModel;

namespace CslaExtremeDemos.Business
{
    public enum CivilStatus : byte
    {
        [Description("<Select status>")]
        Empty = 0,
        [Description("Single")]
        Singleton,
        [Description("Maried")]
        Maried,
        [Description("Divorced")]
        Divorced,
        [Description("Widowed")]
        Widowed
    }
}